### ISSUE TITLE
Accessibility Issue - Timepicker loses focus

### ISSUE THREAD

**Issue by wac2007** [ng-bootstrap/ng-bootstrap#2015]
### Bug description:
When use tab to move to the next field, the focus has lose. Tested in the Demo page.
https://ng-bootstrap.github.io/#/components/timepicker/examples

**Comment by bastienmoulia** [comment]
The focus is still present on the *btn-link* but not visible with bootstrap 4 beta 2.
There is already an issue https://github.com/twbs/bootstrap/issues/24720 and a PR https://github.com/twbs/bootstrap/pull/24723.

**Comment by pkozlowski-opensource** [comment]
twbs/bootstrap#24723 was merged on the Bootstrap side but things still not looking great on our timepicker. We need to see if anything else should be added to Bootstrap or maybe we shell add custom styles to better indicate user's focus.

**Comment by maxokorokov** [comment]
This was fixed a long time ago in #2053 and available in 4.1.3
